lib/canvas_exporting/exporter.rb

```diff
@@ -37,13 +51,13 @@ def export
       if /Error/ =~ result
         render text: result, status: 500
       else
-        send_file @output_tmp_file.path, filename: "#{filename}.#{extension}", type: type
+        send_file @output_file, filename: "#{filename}", type: type
       end
     end
 
     protected
     def tmp_dir
-      Rails.root.join('tmp/highcharts').to_s.tap { |f| FileUtils.mkdir_p f }
+      Rails.root.join('tmp/canvas_charts').to_s.tap { |f| FileUtils.mkdir_p f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.